common/test/java/com/couchbase/lite/LiveQueryTest.java

```diff
@@ -314,7 +314,6 @@ public void testRemovingLiveQuery() throws CouchbaseLiteException, InterruptedEx
     }
```

Comment:
I don't believe this test is testing anything.  Would you either say what it is testing or remove it

Here's the original test.  I don't know how it got turned into what it is now.
https://github.com/couchbase/couchbase-lite-android/blob/4cb39c1c7f69dd6d8e50d75997de7618327215db/android/CouchbaseLite/src/androidTest/java/com/couchbase/lite/LiveQueryTest.java#L62-L141

I gotta tell you, I just spent nearly 15 minutes assuring myself that it wasn't me that made this test useless ;-P

If you can make sense of the original, cool.  If not, I guess we can just dump it.  It hasn't been testing anything at all for at least 2 years.


S